client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -294,34 +294,34 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
         fisrtUser.newInvestigationsCount - secondUser.newInvestigationsCount ||
         fisrtUser.activeInvestigationsCount - secondUser.activeInvestigationsCount
 
-    const fetchAllCountyUsers = () => {
+    const fetchAllCountyUsers = async () => {
         const countyUsersLogger = logger.setup('Getting group users');
         countyUsersLogger.info('requesting the server the connected admin group users', Severity.LOW);
-        axios.get('/users/group')
-            .then((result: any) => {
-                let countyUsers: Map<string, User> = new Map();
-                if (result && result.data) {
-                    result.data.forEach((user: any) => {
-                        countyUsers.set(user.id, {
-                            ...user,
-                            newInvestigationsCount: user.newInvestigationsCount.totalCount,
-                            activeInvestigationsCount: user.activeInvestigationsCount.totalCount,
-                            pauseInvestigationsCount: user.pauseInvestigationsCount.totalCount
-                        })
-                        countyUsers = new Map(Array.from(countyUsers.entries())
-                            .sort((fisrtUser, secondUser) => sortUsersByAvailability(fisrtUser[1], secondUser[1])));
-                    });
-                    countyUsersLogger.info('fetched all the users successfully', Severity.LOW);
-                    setAllUsersOfCurrCounty(countyUsers);
-                } else {
-                    countyUsersLogger.warn('the connected admin doesnt have group users', Severity.MEDIUM);
-                }
-            }).catch(err => {
-                countyUsersLogger.error(err, Severity.HIGH);
-                alertError('לא ניתן היה לשלוף חוקרים');
-            });
+        let countyUsers: Map<string, User> = new Map();
```

Comment:
again, think how to avoid the use of let here - redefine the variable reduce the predictability of the code

Reply:
there is a need to sort the Map Object because of that there is a re-declaration of a new Map but now it's without let :)